src/main/java/seedu/address/model/person/attendance/AttendanceType.java

```diff
@@ -9,7 +9,7 @@ public enum AttendanceType {
     PRESENT,
     ON_LEAVE;
     public static final String MESSAGE_CONSTRAINTS =
-            "Attendance type should only contain alphabets. It should either be 'present', 'late' or 'absent'.";
+            "Attendance type should only either be 'present', 'late' or 'absent'.";
```

Comment:
I think this is an enhancement to the error message, so we'd better not change it?

Reply:
okay!

Reply:
Then I think I'll close this PR! since that's the main change I made